zipline/finance/performance/tracker.py

```diff
@@ -78,41 +78,50 @@ class PerformanceTracker(object):
     """
     Tracks the performance of the algorithm.
     """
-    def __init__(self, sim_params, trading_schedule, env):
+    def __init__(self, sim_params, trading_calendar, env):
         self.sim_params = sim_params
-        self.trading_schedule = trading_schedule
+        self.trading_calendar = trading_calendar
         self.asset_finder = env.asset_finder
         self.treasury_curves = env.treasury_curves
 
         self.period_start = self.sim_params.period_start
         self.period_end = self.sim_params.period_end
         self.last_close = self.sim_params.last_close
-        first_open = self.sim_params.first_open.tz_convert(trading_schedule.tz)
-        self.day = pd.Timestamp(datetime(first_open.year, first_open.month,
-                                         first_open.day), tz='UTC')
-        self.market_open, self.market_close = trading_schedule.start_and_end(
-            self.day
+        first_open = self.sim_params.first_open.tz_convert(trading_calendar.tz)
+
+        self.current_session = pd.Timestamp(
```

Comment:
yes it is, and no, it's not part of the public interface.  I've updated it accordingly.